src/main/webapp/homepage_script.js

```diff
@@ -21,8 +21,8 @@ window.onscroll = stickyControlBar;
 function stickyControlBar() {
     const controlBarWrapper = document.getElementById("control-bar-message-wrapper");
     const taskListDiv = document.getElementById("tasks-list");
```

Comment:
Sorry to nitpick, but these two are also const and it might be best to change them

Reply:
I'm going to have to disagree with this one, if that's alright. These are not primitive types but objects whose properties are going to change. Looking at some of the ToTT resources, they also use const without uppercase on some occasions, so I'd rather keep this in camelCase to avoid readers thinking that the HTML object itself is immutable when only its reference is.

Reply:
If their properties are going to change would it not be better to use "let" or "var" over "const"?

Reply:
I want to keep the variable reference constant so it always points to the same element and doesn't get reassigned to a different element. But either way works, I'll just change this to 'let' to simplify things!